centralstore/redis_metrics_recorder.go

```diff
@@ -61,10 +61,13 @@ func (r *RedisMetricsRecorder) monitor() {
 	for {
 		select {
 		case <-ticker.Chan():
-			allmetrics := r.Metrics.GetAll()
-			conn := r.RedisClient.Get()
-			conn.SetHashTTL("Refinery_Metrics_"+r.prefix, allmetrics, r.reportingFreq*2)
-			conn.Close()
+			// if we don't have a redis client, we can't report metrics to it
```

Comment:
Whoops, well, I had noticed this potential problem and it snuck into this PR. It technically doesn't belong here, but doesn't feel worth it to me to remove it.